platform/src/common/components/AQNumberCard/index.jsx

```diff
@@ -7,23 +7,16 @@ import { setOpenModal, setModalType } from '@/lib/store/services/downloadModal';
 import { fetchRecentMeasurementsData } from '@/lib/store/services/deviceRegistry/RecentMeasurementsSlice';
 import PropTypes from 'prop-types';
 
-// Constants
-const AIR_QUALITY_LEVELS = [
-  { max: 12, text: 'Air Quality is Good', status: 'good' },
-  { max: 35.4, text: 'Air Quality is Moderate', status: 'moderate' },
-  {
-    max: 55.4,
-    text: 'Air Quality is Unhealthy for Sensitive Groups',
-    status: 'unhealthy-sensitive',
-  },
-  { max: 150.4, text: 'Air Quality is Unhealthy', status: 'unhealthy' },
-  {
-    max: 250.4,
-    text: 'Air Quality is Very Unhealthy',
-    status: 'very-unhealthy',
-  },
-  { max: 500, text: 'Air Quality is Hazardous', status: 'hazardous' },
-];
+// Mapping from aqi_category to IconMap keys
```

Comment:
Would this mapping be necessary if we just used the single word color_names? That is:

1. Green
2. Yellow
3. Orange
4. Red
5. Purple
6. Maroon

And then you use those specific color_names to map to whatever you want in the code below. As you do this mapping, you will NOT be using the colors but rather, you will be using their names as KEYS (identifiers) of the different categories.